packages/jaeger-ui/src/components/TracePage/index.js

```diff
@@ -374,18 +385,25 @@ export class TracePageImpl extends React.PureComponent<TracePageProps, TracePage
             />
           )}
         </div>
-        {headerHeight && (
-          <section style={{ paddingTop: headerHeight }}>
-            <TraceTimelineViewer
-              registerAccessors={this._scrollManager.setAccessors}
-              findMatchesIDs={findMatchesIDs}
-              trace={data}
-              updateNextViewRangeTime={this.updateNextViewRangeTime}
-              updateViewRangeTime={this.updateViewRangeTime}
-              viewRange={viewRange}
-            />
-          </section>
-        )}
+        {headerHeight &&
```

Comment:
Fairly minor, but I feel as though this would be clearer if it read:
```
{headerHeight && (
  traceGraphView
    ? <section ...>
      <TraceTimelineViewer ... />
    </section>
    : <section ...>
      <TraceGraph ... />
    </section>
)}

Reply:
Ok changed this to the proposed solution. Looks clearer.